app/javascript/spec/generic-objects-form/__snapshots__/generic-objects-form.spec.js.snap

```diff
@@ -3799,6 +3799,17 @@ exports[`Generic Object Form Component should render adding a new generic object
                                                 />
                                               </label>
                                             </span>
+                                            <div>
+                                              <span
+                                                style={
+                                                  Object {
+                                                    "color": "red",
+                                                  }
+                                                }
+                                              >
+                                                File is too large, maximum allowed size is 5000000 bytes. Current file has undefined byte
```

Comment:
@MelsHyrule I think the original conditional isn't correctly handling an undefined `value.inputFiles[0].size` value as we shouldn't be getting this message is the size is undefined.

Reply:
Merging anyway...I consider this a different bug than the translation one here.

Reply:
Agreed